### PR TITLE
Show progressbar on resync

### PR DESCRIPTION
**Pull request by andreyvEze**

### CODE REVIEW

File: src/components/themed/WalletProgressIcon.js [hunk 1]
```diff
@@ -36,6 +36,11 @@ export class WalletProgressIconComponent extends React.PureComponent<Props, Stat
         this.setState({ isDone: true })
       }, 5000)
     }
+    if (this.props.progress === 0) {
+      setTimeout(() => {
+        this.setState({ isDone: false })
+      }, 5000)
```
Comment (paullinator) [inline src/components/themed/WalletProgressIcon.js:42]: Five seconds seems like overkill. Please set to 500 ms here as well as on line 37

File: src/reducers/scenes/WalletsReducer.js [hunk 2]
```diff
@@ -185,6 +185,14 @@ const walletLoadingProgress = (state = {}, action: Action): $PropertyType<Wallet
       }
     }
 
+    case 'RESET_WALLET_LOADING_PROGRESS': {
+      if (!action.data) throw new Error('Invalid action')
+      return {
+        ...state,
+        [action.data.walletId]: 0
```
Comment (paullinator) [inline src/reducers/scenes/WalletsReducer.js:192]: Progress at zero does not show any progress indicator. Set to 5 instead

**Review by paullinator — CHANGES_REQUESTED** [review]

**Comment by andreyvEze** [comment]
@paullinator just as summary of these changes, e top progress-bar also would animate backwards, is it ok?

**Comment by paullinator** [comment]
> @paullinator just as summary of these changes, top progress-bar also would animate backwards, is it ok?

Sure. And that is fine

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**